desc/equilibrium/equilibrium.py

```diff
@@ -777,7 +785,7 @@ def get_axis(self):
         axis = FourierRZCurve(R_n, Z_n, modes_R, modes_Z, NFP=self.NFP, sym=self.sym)
         return axis
 
-    def compute(
+    def compute(  # noqa: C901
```

Comment:
I can't add tests to see if `eq.compute("something that needs field line grid", grid)` works on this branch. I've merged this `ripple` branch and it works.